compiler/front/main.nim

```diff
@@ -519,6 +519,17 @@ proc mainCommand*(graph: ModuleGraph) =
   when defined(nimDebugUnreportedErrors):
     addExitProc proc = echoAndResetUnreportedErrors(conf)
 
+  when defined(gcOrc):
```

Comment:
would disabling gc be better in `nim.nim`, that way it's restricted to the compiler binary, rather than anything that pulls in `main`?

Reply:
Going by the code, this looks more like the main proc to me as you kinda become the `nim` frontend when you use this, but I don't object moving to `nim.nim` if needed.

Reply:
It's a soft suggestion, just feels symmetric with the change in `nimsuggest.nim`.